knockoff/utilities/ioc.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021-present, Nike, Inc.
```

Comment:
What does ioc stand for?

Reply:
inversion of control which we get with dependency injection